beacon-chain/core/helpers/committee.go

```diff
@@ -451,6 +438,31 @@ func UpdateCommitteeCache(state *pb.BeaconState, epoch uint64) error {
 	return nil
 }
 
+// UpdateProposerIndicesInCache updates proposer indices entry of the committee cache.
+func UpdateProposerIndicesInCache(state *pb.BeaconState, epoch uint64) error {
+	if !featureconfig.Get().EnableProposerIndexCache {
+		return nil
+	}
+
```

Comment:
```suggestion
```

Reply:
The extra line is nice for separation clarity. I propose we keep it

Reply:
fair